src/jit/lsra.cpp

```diff
@@ -5604,6 +5604,20 @@ regNumber LinearScan::tryAllocateFreeReg(Interval* currentInterval, RefPosition*
             else if ((bestScore & UNASSIGNED) != 0 && intervalToUnassign != nullptr)
             {
                 availablePhysRegInterval->previousInterval = intervalToUnassign;
+#ifdef _TARGET_ARM_
+                // TODO-ARM-Throughput: For ARM,  we don't have to keep a same previous interval in two RegRecords,
+                // if we can use a interval from one RegRecord. But I'm not sure yet.
+                // Please take a look at unassignPhysReg().
```

Comment:
One more suggestion: You might update this comment to add that it should not be necessary because we will always manage the register assignment of `TYP_DOUBLE` intervals together.